frontend/javascripts/oxalis/view/components/setting_input_views.js

```diff
@@ -507,15 +507,9 @@ export class ColorSetting extends React.PureComponent<ColorSettingPropTypes> {
     style = style || {};
     return (
       <div
-        id="color-picker-wrapper"
```

Comment:
As each of the bounding boxes rendered in the bounding box tab has its own diff, the name `color-picker-wrapper` is not unique and therefore a class